k8s/inventory/values-prod.yaml

```
@@ -14,7 +14,7 @@ service:
 ingress:
   enabled: false
 nodeSelector: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)